css/stylesheet.css

```diff
@@ -210,3 +210,24 @@ img {
   text-decoration: underline;
   text-decoration-color: red;
 }
+
+/* Scroll to top */
+#scrollToTop {
+    display: none;
+    position: fixed;
+    bottom: 20px;
+    right: 30px;
+    z-index: 5;
+    opacity: 0.5;
+    border: none;
+    outline: none;
+    background-color: aqua;
+    color: blue;
+    cursor: pointer;
+    padding: 15px;
+    border-radius: 10px;
+}
+#scrollToTop:hover {

```

Comment:
Is there a reason why you duplicate the code? You can create a style sheet and include it in both documents. Or should if look differently?

Reply:
@niccokunzmann I thought that one css file for each page should be there but I think a new stylesheet can be made. I will change the commit.